src/frontend/src/components/ProposedSolutionForm.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import PageBlock from '../layouts/PageBlock';
+import { Button, Col, Form, InputGroup, Row } from 'react-bootstrap';
+import * as yup from 'yup';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+interface ProposedSolution {
+  description: string;
+  budget: number;
+  timeline: number;
+  scope: string;
```

Comment:
Same idea as budget applies here. “How much additional timeline is needed?” And “What changes to the scope does this entail?”

Reply:
same idea as my comment then